zanata-war/src/main/java/org/zanata/action/DashboardAction.java

```diff
@@ -59,7 +60,8 @@
 
 @Name("dashboardAction")
 @Scope(ScopeType.PAGE)
-@Restrict("#{identity.loggedIn}")
+@ZanataSecured
+@CheckLoggedIn
```

Comment:
>>    * Should we put @Interceptors(ZanataSecurityInterceptor.class) on the @interface CheckLoggedIn too?

> No need to do that.

But it would guarantee that a class-level `@CheckLoggedIn` would work, would it not?  Without having to remember the `@ZanataSecured`.